lti_consumer/plugin/views.py

```diff
@@ -415,3 +418,130 @@ def scores(self, request, *args, **kwargs):  # pylint: disable=unused-argument
             status=status.HTTP_201_CREATED,
             headers=headers
         )
+
+
+class LtiNrpsContextMembershipViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    LTI NRPS Context Membership Service endpoint.
+    """
+
+    # Custom permission classes for LTI APIs
+    authentication_classes = [Lti1p3ApiAuthentication]
+    permission_classes = [LtiNrpsContextMembershipsPermissions]
+
+    # Renderer classes to accept LTI NRPS content types
+    renderer_classes = [
+        MembershipResultRenderer,
+    ]
+
+    serializer_class = LtiNrpsContextMembershipSerializer
```

Comment:
Pagination and the DRF request limit don't seem to be working for me. Can you test if the `PAGE_SIZE` and `limit` parameters work for you? They are not working for me.

I think it's because you're not passing the `limit` and page parameters to the LMS retrieval method.

Reply:
This has been fixed now. Thanks for reporting the issue.